src/Provider/Provider.Web/Controllers/SearchHelperController.cs

```diff
@@ -0,0 +1,25 @@
+using System.Threading.Tasks;
+using Esfa.Recruit.Provider.Web.Configuration.Routing;
+using Microsoft.AspNetCore.Mvc;
+using Esfa.Recruit.Provider.Web.Orchestrators;
+using Esfa.Recruit.Provider.Web.RouteModel;
+
+namespace Esfa.Recruit.Provider.Web.Controllers
+{
+    [Route(RoutePaths.AccountRoutePath)]
+    public class SearchHelperController : Controller
+    {
+        private readonly SearchHelperOrchestrator _orchestrator;
+        public SearchHelperController(SearchHelperOrchestrator orchestrator)
+        {
+            _orchestrator = orchestrator;
+        }
+
+        [HttpGet("search-helper")]
```

Comment:
This is for ajax call only, do we still need route name? 

It brings back autocomplete list (not a filter) hence I used helper, could have used SearchAutoCompleteList does that work?

Reply:
Yes for convention reasons as well as it will help in one of the suggestions I have made below where you need to resolve the AJAX endpoint url you are calling from script Java.

@mgwilliam can help with naming the url portion of this endpoint, he is great at naming.

Reply:
fixed